dashboard/app/controllers/lessons_controller.rb

```diff
@@ -24,7 +24,7 @@ def show
   # GET /lessons/1/edit
   def edit
     @lesson_data = @lesson.summarize_for_lesson_edit
-    @related_lessons = @lesson.summarize_related_lessons
+    @related_lessons = []
```

Comment:
nit: can you add a comment explaining why this is just an empty list?

Reply:
I've added a brief comment. Please let me know if you'd like to see more.

Reply:
Looks good! Thanks!